cmd/cstor-pool-mgmt/controller/pool-controller/new_pool_controller.go

```diff
@@ -113,6 +116,11 @@ func NewCStorPoolController(
 			if IsDeletionFailedBefore(cStorPool) || IsErrorDuplicate(cStorPool) {
 				return
 			}
+			if cStorPool.Labels[string(apis.OpenEBSVersionKey)] != curVersion {
+				glog.Infof("csp %s is not reconciled current version of controller: %s csp version: %s",
```

Comment:
These logs are not understood. We need to reword.
These logs can also be removed or logged at V(4) level.

Reply:
Updated the log message PTAL